theseus_gui/src/helpers/metadata.js

```diff
@@ -37,3 +37,11 @@ export async function get_neoforge_versions() {
   console.log('Getting neoforge versions', c)
   return c
 }
+
+// Gets the current launcher version
+// Returns String
+export async function get_launcher_version() {
+  const c = await invoke('plugin:metadata|metadata_get_launcher_version')
```

Comment:
does this logic need to use `invoke` and find an env var every single time the settings page is opened?

Reply:
this could easily be transferred to a simple `package.json` `version` import or just an env read within javascript rather than getting it from rust and tauri. that would increase efficiency drastically

Reply:
nope, no reason...

for some reason, i was certain that the version in the `package.json` wasn't correct... 

thanks for pointing it out